packages/button/README.md

```diff
@@ -26,6 +26,8 @@ Each page will likely have one primary button. Any remaining calls-to-action are
 
 **Processing**: When user clicks or taps a button and an API request is initiated we should provide feedback so they know their action was received.
 
+**Hidden:** Hide a button when there is no way a user can interact with the button.
+
 ## Using buttons as links
 
 Sometimes there is a button that is actually a link to somewhere else. If the appropriate props are set, the button is rendered as an anchor tag, but it is styled like a button.
```

Comment:
Is this part of usage? 
if so, appropriate when action is a secondary action and as per certain design patterns like Empty States.

Reply:
Yes, this is part of button usage guidelines. I don't understand what changes you're suggesting when you say "appropriate when action is a secondary action and as per certain design patterns like Empty States"

Reply:
Excuse me, I am saying we may want to clarify when appropriate to use this secondary style of button. If we cover this already in buttons, OK. If this is the only ref to secondary buttons, we should communicate that used for this nav case but also non-primary actions (e.g,  Download All Logs).